mqtt-format/src/v3/packet.rs

```diff
@@ -26,62 +27,179 @@ use super::{
     MSResult,
 };
 
+#[cfg_attr(feature = "yoke", derive(yoke::Yokeable))]
```

Comment:
This feature does not exist in `Cargo.toml` of the crate.

Reply:
Added a note!